pytest_fixtures/component/satellite_auth.py

```diff
@@ -45,7 +45,7 @@ def ldap_cleanup():
 def ad_data():
     supported_server_versions = ['2016', '2019']
 
-    def _ad_data(version='2016'):
+    def _ad_data(version='2019'):
```

Comment:
You might need to update auth fixtures as this could change this condition `ad_data('2019') if '2019' in auth_type else ad_data()`?

Reply:
I think as it is now, the fixture returns 2019 by default. So if anything, that condition can be dropped. I didn't, however, because I want to make only minimal changes to migrate the instance and catch potential problems caused by it.

Reply:
I think if we are not supporting the 2016 AD server and guessing this is discussed with programme members and the product manager.  I think it's a wise decision to remove the dead code related to the 2016 AD server from robottelo, given that we're no longer supporting it. Although you wanted to keep changes minimal, we were planning to remove this code anyway.

I'd also like to point out that two tests, specifically https://github.com/SatelliteQE/robottelo/blob/8580597f43533d2546c4a6575ea813f8a262a9a1/tests/foreman/destructive/test_ldap_authentication.py#L220 and https://github.com/SatelliteQE/robottelo/blob/867c00dfebe77c2b0fe517ab3319d82732358370/tests/foreman/ui/test_ldap_authentication.py#L562, are currently using both the 2016 and 2019 AD servers. This means that these tests will run twice with the 2019 AD server, which is not the correct behavior.